### DISCUSSION TITLE
P-tuning后如何对模型进行评估、验证？

### DISCUSSION THREAD

**Discussion by gxc199912** [zai-org/ChatGLM3#509] [Q&A]
在低版本chat-glm中提供了train.sh以及evaluate.sh文件进行训练和验证，在chat-glm3中仅有finetune文件进行微调，请问微调后如何进行验证？

**Comment by zRzRzRzRzRzRzR** [comment]
目前没有提供验证代码，如果你用的是 llama-factory可以用他们自带的那套验证方法
#253 微调问题请在这里提出
